frontend/components/utils/AnimatedNumber.vue

```diff
@@ -1,10 +1,12 @@
 <template>
-    <animated-number
-        v-if="isValidNumber"
-        :value="formatValue(value)"
-        :format-value="formatValue"
-        :duration="duration"
-    />
+    <span v-if="isValidNumber"
+        ><animated-number
+            v-if="displayFullNumber"
+            :value="formatValue(value)"
+            :format-value="formatValue"
+            :duration="duration"
+        /><span v-else> under 0.00001 </span></span
```

Comment:
- Can you please pass `0.00001` as a value to `animated-number`, so it will be properly animated (since we're doing it inside `AnimatedNumber` I would expect it to keep being animated)
- Can you please add negative value as a story (so we can see how it works with the `-`)

Reply:
> * Can you please add negative value as a story (so we can see how it works with the `-`)

I added support for negative numbers and added stories for them as well!